packages/react-instantsearch-core/src/core/createConnector.tsx

```diff
@@ -72,6 +77,15 @@ type ConnectorDescription = {
   defaultProps?: {};
 };
 
+type ConnectorProps = {
+  contextValue: InstantSearchContext;
+  indexContextValue: IndexContext;
```

Comment:
This value is optional. When the component is not outside of  the multi-index context we don't have a value.